05_iCommands_Handson_User-Training.md

```diff
@@ -92,83 +89,91 @@ Let’s create a collection in iRODS and name it “test”.
-
-The -r option is used for collections. Now we can remove test1 collection.
+To remove one or more data objects or collections from iRODS space we use `irm` (again, with the `-r` for collections). However, once we execute this command the items are by default moved first to the trash collection (/yourZone/trash) unless the `-f` option is used.
+Let's can remove test1 collection.
 
 ```sh
 irm -r test1
 ```
 
-**Note:** All the collections and data objects that are deleted move to the trash collection. They are permanently cleaned when they are older than 15 days. Or the `irmtrash` command should be used to delete data-objects and collections in the trash collection instanly. 
+**Note:** All the collections and data objects that are deleted move to the trash collection. They are permanently cleaned when they are older than 15 days. Alternatively, the `irmtrash` command may be used to delete data objects and collections in the trash collection instantly. 
 
-In CLI shells, you can print the contents of a file with the `cat` command. In iRODS, we can do the same with the command `istream` and the option `read`.
-The following would print the contents of the file test1 to the terminal:
+We can create empty files with `itouch` and write them with `istream` followed by `write`:
 
 ```sh
-istream read test1
+itouch test1.txt
+echo 'hello' | istream write -a test1.txt
 ```
 
-The same command can be used to redirect standard input and add it to a data object in iRODS.
-For example, to append the word 'hello' to the end of our file 'test1':
+In CLI shells, you can normally print the contents of a file with the `cat` command. In iRODS we can do the same with the command `istream` and the option `read`.
+The following would print the contents of the file test1.txt to the terminal:
 
 ```sh
-echo 'hello' | istream write -a test1 
+istream read test1.txt
 ```
 
 ### Functional iCommands
 
 With the commands in this section, we will do functional data operations like data uploading/downloading, access control and verifying/synchronizing data.
-This constitutes the basis of data management in iRODS.
+This constitutes the **basis of data management in iRODS**.
 
 #### Data Upload
-We can store a file into iRODS. If the destination data object or collection are not provided, the current iRODS collection and the input file name are used.
 
-To upload data into iRODS we should have a data object in our VSC system. Thus we should first create a data file in our Linux home directory. 
+We can store a file into iRODS with `iput local [irods]`: If the destination data object or collection (`irods`) are not provided, the current iRODS collection and the input file name are used.
+
+To upload data into iRODS we should have a data object in our VSC system. For example, let's create a data file in our Linux home directory. 
```

Comment:
Users should never create files in their $VSC home directory. It's better to use their data directory ($VSC_DATA) instead.

Reply:
Would it be ok to just replace "our VSC system" with "our local system"? Because you don't necessarily access iRODS from VSC, right?

Reply:
Yes, that would be an option. However, I'd especially replace 'in our Linux home directory' with 'in our current directory'.
That would already solve the problem.